src/isotools/_transcriptome_io.py

```diff
@@ -374,8 +386,10 @@ def add_sample_from_bam(self, fn, sample_name=None, barcode_file=None, fuzzy_jun
                                         [snd_align[0], snd_align[2], snd_exons, aligned_part(snd_cigartuples, snd_align[2] == '-'), None])
                                     # logging.debug(chimeric[read.query_name])
                         continue
+                    
+                    # skipping low-quality alignments
                     try:
-                        # if edit distance becomes large relative to read length, skip the alignment
+                        # if edit distance (tags['NM']) becomes large relative to read length (read.query_length), skip the alignment because it has a low alignment fraction
```

Comment:
I don't think referencing `read.query_length` is necessary. That should be clear from context.
Maybe reword to 
> Skip alignments with a low alignment fraction, i.e. large edit distance (tags['NM']) relative to read length